src/components/cv-breadcrumb/_cv-breadcrumb-item-skeleton.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="bx--breadcrumb-item">
+    <a href="/#" class="bx--link">
```

Comment:
This should be disabled (as other active content e.g. button in CvAccordionItemSkeleton).

Use the following for link
aria-disabled tabindex="-1"

The same for buttons, input etc if setting disabled has other visual consequences.

Reply:
Looks like it's enough to remove its `href` attribute to fix this. Is it ok?